src/helpers/filterExpensesByQuery.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const filterByUserId = (expenses, userId) => {
+  if (userId) {
+    return expenses.filter(item => item.userId === Number(userId));
+  }
+
+  return expenses;
+};
+
+const filterByCategories = (expenses, categories) => {
+  if (categories) {
+    return expenses.filter(({ category }) => categories.includes(category));
```

Comment:
if there are no `categories` in search params we will get `undefined`
if only 1 `categories` we will get just value (....?categories=1)
And if we have many categories we will get array of values (....?categories=1&categories=2&categories=3)

![image](https://github.com/mate-academy/node_accounting-app-v2/assets/86789161/ea010564-92a2-4116-9fcf-a66739d6d01f)

Your code will crash if you have only 1 category